packages/pwa-kit-runtime/src/utils/logger.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+const LOG_LEVELS = ['debug', 'info', 'warn', 'error']
```

Comment:
Based on your code, the order of this array is important. You might want to note this for future developers that might want to come in an order this alphabetically.